containers/dotnet/definition-manifest.json

```diff
@@ -7,7 +7,11 @@
 		"tags": [
 			"dotnet:${VERSION}-${VARIANT}"
 		],
+		"architectures": ["linux/amd64", "linux/arm64"],
```

Comment:
I just tried and building an arm64 image for the focal variants fails on macOS and will also in our CI.  You'll see an error about GPG key missing and this is due to a curl segfault that has no resolution.

See python-3 for how this was set up to change architecture by variant.